cicd/sample_log_dataset_queries.csv

```diff
@@ -0,0 +1,116 @@
+*,now-1d,now,*,total,gt,500,Pipe QL
+app_name=Zebracould,now-1d,now,*,total,gt,0,Pipe QL
+batch=batch-*,now-1d,now,*,total,gt,100,Pipe QL
+min(zip) groupby batch,now-1d,now,*,total,,100,Pipe QL
+Sunday,now-1d,now,*,total,gt,1,Pipe QL
+arvidlesch@hilll.org,now-1d,now,*,total,gt,0,Pipe QL
+http_status>400,now-1d,now,*,total,gt,0,Pipe QL
+cardinality(user_email),now-1d,now,total,*,,1,Pipe QL
+"""https://picsum.photos/130/268""",now-1d,now,*,total,gt,0,Pipe QL
+latency<9539889 AND app_name=Koalahad,now-1d,now,*,total,gt,0,Pipe QL
+"""MintCream""",now-1d,now,*,total,gt,100,Pipe QL
+"(MintCream AND (""Thursday"" OR ""Monday""))",now-1d,now,*,total,gt,20,Pipe QL
+"(MintCream AND (""Thursday"" OR ""Monday"")) | min(latency) groupby state",now-1d,now,*,total,gt,20,Pipe QL
+latency<10000 AND latitude>0 AND longitude > 0,now-1d,now,*,total,gt,10,Pipe QL
+Wisconsin,now-1d,now,*,total,gt,100,Pipe QL
+female,now-1d,now,*,total,gt,100,Pipe QL
```

Comment:
I think the corresponding line in ingest.csv and restart.csv for this line `female,now-1d,now,*,total,gt,100,Pipe QL` is `404 OR 500,now-1d,now,*,total,gt,100,Pipe QL`. So could you add an `OR` to this search, or correct me if I'm wrong?